src/docs/09_design_decisions.adoc

```diff
@@ -4,7 +4,12 @@
 === Route specification
 We follow the https://github.com/Arquisoft/viadeSpec[Viade Specficiation] to set a format to manage the routes, so that interoperability can be achieved.
 
-
 === Leaflet
 We chose this library because it is open source, free, and matches the philosophy of Solid.
 
+=== Template
+At the begining of the course, we had to choose using the Inrupt Generator or not. After some discussion, we decided not to use it in order to learn React from the basics, as we considered not only important the SOLID challenge, but also learning a new technology.
+This decision carried us some disadvantages, for example we lost some time learning React, but we consider it worthy because of the knowledge gained.
+
+=== External Manual
+As it might be difficult for tech newbies the SOLID concepts or our application, we have created an external webpage that will work as a manual of our appllication. This was done in another webpage so the actual application was separated from this manual. This external manual will be hosted in https://lamasumas.github.io/Solid/[HERE].
```

Comment:
You have a typo on the first line, _SOLID concepts *of* our application_. Also another typo on the second line: _that will work as a manual of our *application*._